tests/InputFiles/GeneralizedHarmonic/BinaryBlackHole.yaml

```diff
@@ -32,7 +32,7 @@ Evolution:
         SafetyFactor: 0.95
   TimeStepper:
     AdamsBashforth:
-      Order: 5
+      Order: 4
```

Comment:
This input file is just parsed, not run, so should we just delete it @knelli2 @geoffrey4444 ? The `Inspiral.yaml` above is already parsed in a unit test, so I don't think this input file adds anything useful, and it would just be tedious to update.

Reply:
I also think we might be able to delete this, but consider that all our other executables have their "example" input files here in the test directory. I imagine users who run our other executables would look here initially for a BBH and not see it. Maybe that's even what we want here since we want people to use the pipeline

Reply:
so should I delete this file in this pr?

Reply:
@nilsvu and I say yes (met in person). If we ever need it back we can just re-add it. The important thing is that the BBH is parsed somehow which is already tested by the pipeline.